### ISSUE TITLE
v0.6.0 crashes Julia on upper left icon click

### ISSUE THREAD

**Issue by tro3** [timholy/ProfileView.jl#129]
When trying out v0.6.0, simply running the tutorial demo, I accidentally hit the upper leftmost icon (print, I think) and crashed not only the window, but Julia itself.

```julia
julia> @profview profile_test(10);

[ok here, until I hit the upper leftmost icon in the GTK window]

julia> FATAL ERROR: Gtk state corrupted by error thrown in a callback:
ERROR: UndefVarError: c not defined
Stacktrace:
 [1] open_cb(::Ptr{Gtk.GLib.GObject}, ::Tuple{Gtk.GtkCanvas,Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}}) at C:\Users\Trey Roessig\.julia\packages\ProfileView\HysnR\src\ProfileView.jl:189
 [2] (::Gtk.var"#237#238")() at C:\Users\Trey Roessig\.julia\packages\Gtk\5KVNb\src\events.jl:2
 [3] g_sigatom(::Any) at C:\Users\Trey Roessig\.julia\packages\Gtk\5KVNb\src\GLib\signals.jl:176
 [4] gtk_main() at C:\Users\Trey Roessig\.julia\packages\Gtk\5KVNb\src\events.jl:1

[Julia dies - back to shell prompt]
```

My version info
```julia
julia> versioninfo()
Julia Version 1.3.1
Commit 2d5741174c (2019-12-30 21:36 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-4500U CPU @ 1.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, haswell)
Environment:
  JULIA_NUM_THREADS = 4
```

Hope this helps.  Not a big deal for me

**Comment by tro3** [comment]
Hmm.  The second time loading, with a fresh recompilation, I now get 

```
┌ Warning: You are using Gtk on Windows which is currently not recommended. Your REPL/IDE and anything depending on task switches will become sluggish and much slower (up to ~85x slower).
└ @ Gtk C:\Users\Trey Roessig\.julia\packages\Gtk\5KVNb\src\Gtk.jl:74
```

Perhaps any complaints of crashes on PC's are moot, in which case you can close this out.

**Comment by timholy** [comment]
No, Gtk should work, it just slows responsivity of the REPL. See https://github.com/JuliaGraphics/Gtk.jl/issues/325 and possible fix in https://github.com/JuliaGraphics/Gtk.jl/pull/467#issuecomment-564593402.

Will be fixed by #131.
